docs/source/model/dependencies.rst

```diff
@@ -0,0 +1,446 @@
+Managing Dependencies in MLflow Model
+=====================================
+
+`MLflow Model <../models.html>`_ is a standard format that packages a machine learning model with its dependencies and other metadata.
+Building a model with its dependencies allows for reproducibility and portability across a variety of platforms and tools.
+
+When you create an MLflow model using the `MLflow Tracking APIs <../tracking.html>`_, for instance, :py:func:`mlflow.pytorch.log_model`,
+MLflow automatically infers the required dependencies to your model flavor and records them as a part of Model metadata. Then when you
+serve the model for prediction, MLflow automatically installs the dependencies to the environment. Thus, normally you don't need to
+worry about managing dependencies in MLflow Model.
+
+However, in some cases, you may need to add or modify some dependencies. This page provides a high-level description of how MLflow manages
+dependencies and guidance for how to customize dependencies for your use case.
+
+.. contents:: Table of Contents
```

Comment:
note: Adding table of contents as the side bar doesn't show indexes at this level.

Reply:
Good idea.